tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -475,13 +476,12 @@ def test_positive_update_name(self):
             with self.subTest(new_name):
                 ContentView.filter.update({
                     'content-view-id': self.content_view['id'],
-                    'name': cvf_name,
+                    'id': cvf['filter-id'],
                     'new-name': new_name,
-                    'organization-id': self.org['id'],
```

Comment:
As per BZ1388642 `organization-id` was accidentally removed. As we have separate test for that specific case, i'm unblocking this test by using entity id instead of name+org.